language/cloud-client/v1/snippets.py

```diff
@@ -51,14 +54,20 @@ def sentiment_text(text):
     print('Score: {}'.format(sentiment.score))
     print('Magnitude: {}'.format(sentiment.magnitude))
     # [END language_python_migration_sentiment_text]
-# [END language_sentiment_text]
+    # [END language_sentiment_text]
 
 
-# [START language_sentiment_gcs]
 def sentiment_file(gcs_uri):
+    # [START language_sentiment_gcs]
+    from google.cloud import language
+    from google.cloud.language import enums
+    from google.cloud.language import types
     """Detects sentiment in the file located in Google Cloud Storage."""
     client = language.LanguageServiceClient()
 
+    # TODO(developer): Uncomment the following line to run this code.
+    # gcs_uri = 'The gcs uri to your file, e.g. gs://my_bucket/my_file'
```

Comment:
This URI could be defined here if you put the file in our sample data GCS bucket - gs://cloud-samples-data. This way the user has an easily accessible test file and the snippet tests don't have to upload a file every time. Let me know if you don't have permissions to that project and I'll add you.